Functions.Rmd

```diff
@@ -77,17 +77,17 @@ Answer the following questions to see if you can safely skip this chapter. You c
 
 ## Function fundamentals
 
-To understand functions in R you need to internalise two important ideas. Functions:
+To understand functions in R you need to internalise two important ideas:
```

Comment:
You have no idea how long I wrestled with these sentences. I really like your tweaks!